components/content/inspira/examples/LinkPreviewDemo.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="flex justify-center items-center h-[40rem] flex-col px-4">
+    <p class="text-neutral-500 dark:text-neutral-400 text-xl md:text-3xl max-w-3xl mx-auto mb-10">
+      <LinkPreview url="https://tailwindcss.com" class="font-bold"> Tailwind CSS </LinkPreview>
+      and
+      <LinkPreview url="https://motion.vueuse.org/" class="font-bold"> @vueuse/motion </LinkPreview>
+      are a great way to build modern websites.
+    </p>
+    <p class="text-neutral-500 dark:text-neutral-400 text-xl md:text-3xl max-w-3xl mx-auto">
+      Visit my
+      <LinkPreview url="https://github.com/atif0075" :width="400" :height="200">
```

Comment:
Can you change this text to something like Visit Inspira UI (https://inspira-ui.com) for more cool components?